packages/lodestar-validator/src/services/attestation.ts

```diff
@@ -35,17 +35,16 @@ import {ISlashingProtection} from "../slashingProtection";
 import {IAttesterDuty} from "../types";
 import {isValidatorAggregator} from "../util/aggregator";
 import {abortableTimeout} from "../util/misc";
-import {getAggregationBits, getAggregatorModulo} from "./utils";
+import {getAggregationBits, getAggregatorModulo, getPubKeyIndex} from "./utils";
 
 export class AttestationService {
   private readonly config: IBeaconConfig;
   private readonly provider: IApiClient;
-  // order is important
-  private readonly secretKeys: SecretKey[] = [];
-  // order is important
-  private readonly publicKeys: BLSPubkey[] = [];
-  // order is important
-  private readonly validators: (ValidatorResponse | null)[] = [];
+  private readonly validatorClients: {
```

Comment:
```suggestion
  private readonly validators: {
```
1 validator client process can host multiple validators